roles/picksome/files/picksome.php

```diff
@@ -0,0 +1,67 @@
+<?php
+# While this next line should be taken care of in extenions, and indeed
+# it is, because this extension is loaded via wfLoadExtension, and because
+# Collection is loaded the old way, the sidebar before output is called
+# preferring Collection, placing it higher on the siderbar than PickSome.
+#
+# This was requested to be changed, and the most straightforward way is
```

Comment:
I think it would be worth being more verbose here, for the reader who doesn't have as much context as you do.